crs_perception/src/nodes/test_localize_to_part.cpp

```diff
@@ -0,0 +1,96 @@
+#include <iostream>
+#include <pcl/io/pcd_io.h>
+#include <pcl/point_types.h>
+#include <pcl/conversions.h>
+#include <pcl_conversions/pcl_conversions.h>
+#include <pcl/common/transforms.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+
+#include <geometry_msgs/msg/transform_stamped.hpp>
+#include <crs_msgs/srv/load_part.hpp>
+#include <crs_msgs/srv/localize_to_part.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+int main(int argc, char **argv) {
+  rclcpp::init(argc, argv);
+  std::shared_ptr<rclcpp::Node> node = rclcpp::Node::make_shared("test_localize_to_part");
+
+  pcl::PointCloud<pcl::PointXYZ>::Ptr point_cloud = boost::make_shared<pcl::PointCloud<pcl::PointXYZ> >();
+  //if (pcl::io::loadPCDFile<pcl::PointXYZ>("/home/ayoungs/Downloads/737_fc.pcd", *point_cloud) == -1)
+  if (pcl::io::loadPCDFile<pcl::PointXYZ>("/home/ayoungs/workspaces/crs/src/test.pcd", *point_cloud) == -1)
```

Comment:
Avoid hardcoded paths

Reply:
I don't plan on keeping this tester around, however, if I do I will update this properly to use parameters.

Reply:
Sounds good